poolmgr/gpm.go

```diff
@@ -157,8 +157,8 @@ func (gpm *GenericPoolManager) eagerPoolCreator() {
 		// creating pools for envs that are actually used by functions.  Also we might want
 		// to keep these eagerly created pools smaller than the ones created when there are
 		// actual function calls.
-		for _, env := range envs {
-			_, err := gpm.GetPool(&env)
```

Comment:
@life1347 
`Get Pool` seems to be synchronized function call, so in the loop, every envs item is copied to `env`, get the address and then send to the function, so it should work fine although the address of `env` is the same?